src/Filter.tsx

```
@@ -1,240 +1,233 @@
-            } else if (filterType === GADimensionFilterType.STRING) {
-              const stringFilter = {
-                matchType: GAStringFilterMatchType.MATCH_TYPE_UNSPECIFIED,
-                caseSensitive: true,
-                value: targetData.stringFilter?.value || ''
-              } as GAStringFilter
-              targetData.stringFilter = stringFilter
-            }
-          }
+        case GADimensionFilterType.IN_LIST:
+          newFilter.inListFilter = { values: [], caseSensitive: false };
+          delete newFilter.stringFilter;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)